src/main/java/fi/metatavu/famifarm/packings/CutPackingController.java

```diff
@@ -0,0 +1,161 @@
+     * @param gutterHoleCount gutter hole count
+     * @param storageCondition storage condition
+     * @param creatorId the id of the user who is creating this cut packing
+     */
+    public CutPacking create (
+            UUID productId,
+            UUID productionLineId,
+            double weight,
+            OffsetDateTime sowingDay,
+            OffsetDateTime cuttingDay,
+            String producer,
+            String contactInformation,
+            int gutterCount,
+            int gutterHoleCount,
+            String storageCondition,
+            UUID creatorId) throws CutPackingInvalidParametersException {
+
+        Product product = productDAO.findById(productId);
+        ProductionLine productionLine = productionLineDAO.findById(productionLineId);
+
+        if (product == null) {
+            throw new CutPackingInvalidParametersException("Product with id " + productId + " not found!");
+        }
+
+        if (productionLine == null) {
+            throw new CutPackingInvalidParametersException("Production line with id " + productionLineId + " not found!");
+        }
+
+        return cutPackingDAO.create(UUID.randomUUID(), product, productionLine, weight, sowingDay, cuttingDay, producer, contactInformation, gutterCount, gutterHoleCount, storageCondition, creatorId);
+    }
+
+    /**
+     * Updates a cut packing
+     *
+     * @param cutPacking a cut packing to update
+     * @param productId the id of a new product
+     * @param productionLineId the id of a new production line
+     * @param weight a new weight
+     * @param sowingDay a new sowing day
+     * @param cuttingDay a new cutting day
+     * @param producer a new producer
+     * @param contactInformation a new contact information
+     * @param gutterCount a new gutter count
+     * @param gutterHoleCount a new gutter count
+     * @param storageCondition a new storage condition
+     * @param modifierId the id of the user who is modifying this cut packing
+     *
+     * @return a cut packing to update
+     */
+    public CutPacking update (
+            CutPacking cutPacking,
+            UUID productId,
+            UUID productionLineId,
+            double weight,
+            OffsetDateTime sowingDay,
+            OffsetDateTime cuttingDay,
+            String producer,
+            String contactInformation,
+            int gutterCount,
+            int gutterHoleCount,
+            String storageCondition,
+            UUID modifierId) throws CutPackingInvalidParametersException {
+
+        Product product = productDAO.findById(productId);
+        ProductionLine productionLine = productionLineDAO.findById(productionLineId);
+
+        if (product == null) {
+            throw new CutPackingInvalidParametersException("Product with id " + productId + " not found!");
+        }
+
+        if (productionLine == null) {
+            throw new CutPackingInvalidParametersException("Production line with id " + productionLineId + " not found!");
```

Comment:
Use formatter instead of concat i.e. String.format("Product with id %s not found!", productId)